modules/api/src/main/java/org/eclipse/xpanse/api/OrchestratorApi.java

```diff
@@ -122,7 +144,7 @@ public List<ServiceStatus> services() {
      * @param managedServiceName name of managed service
      * @return response
      */
-    @PostMapping("/start/{managedServiceName}")
+    @PostMapping("/service/{managedServiceName}")
```

Comment:
now since this API has methods for ISV and end user both, we can consider add tags to the services

@Api(tags = "Service Provider or Service Vendor)
@Api(tags = "End User or Service Consumer")

With this the services should be ideally grouped in the open api json and also the swagger UI